### DISCUSSION TITLE
Please make Oxygenite More Useful

### DISCUSSION THREAD

**Discussion by gerakis** [FakeFishGames/Barotrauma#7868] [Ideas and suggestions]
This is by far the least useful of the alien materials.

Liquid oxygenite: CPR for people too bad at medicine to perform CPR, otherwise useless. Useless if you have a medic, which is almost always.

Oxygenite tank: too rare and expensive to ever be used. Doesnt last forever in masks or plasma cutters, doesnt stack with regular tanks. Cannot be recharged. Speed boost is the only real advantage and is marginal. Barely lasts longer than a regular tank. Considering regular oxygen tanks work well enough, what is the point of this? 

Deusizine component: this is its best use, but you can still just buy liquid oxygenite in stores so its not that important to collect it from ruins

Infinite Oxygen Source (if left on ground): incredibly explosive and shock sensitive, so if you rely on it to provide oxygen for your ship you will go up like the fourth of july as soon as you meet a hammerhead. Does not provide oxygen inside containers. No way to render it safe for shocks and also provide oxygen. Too dangerous to ever use for this.

Shards as  plasma cutter fuel: the only real use, but you patched this out for some reason.

This is overall pretty underwhelming. I dont even bother picking it up any more unless I have a lot of spare inventory space. Even then I just load it into railgun shells like ghetto C-4. Other than its use in making deusizine its close to worthless.

I would recommend making some way of providing permanent electricity-free oxygen without the risk of explosion. Like some component you can make called "Oxygenite Generator" that you mount to the wall like a component. Or perhaps reduce the power bill of the oxygen generator by 20% for every shard you add to it. 

Also either make oxygenite tanks refillable, or just make them last forever.

**Comment by TheDarkBox** [comment]
I like the idea of being able to improve the Oxygen Generator with this.

**Comment by Ek-Videogames** [comment]
Oxygenite is a medicine's main problem is that the health lost due to organ damage almost completely nullifies any health that you would have gained from oxygen restoration. 
And - of course - the only medicines that can heal organ damage reduce their oxygen as well, leading to an impossible `"old lady who swallowed a fly"` type paradox.
